src/models/vehicle/Vehicle.jsx

```diff
@@ -114,16 +115,19 @@ function VehicleAudio() {
   const brakeAudio = useRef()
   const [sound, maxSpeed] = useStore((state) => [state.sound, state.vehicleConfig.maxSpeed])
 
+  let rpmTarget = 0
   let controls
   let speed = 0
-  useFrame(() => {
+  useFrame((state, delta) => {
     speed = mutation.speed
     controls = useStore.getState().controls
 
     boostAudio.current.setVolume(sound ? (controls.boost ? Math.pow(speed / maxSpeed, 1.5) + 0.5 : 0) * 5 : 0)
     boostAudio.current.setPlaybackRate(Math.pow(speed / maxSpeed, 1.5) + 0.5)
-    engineAudio.current.setVolume(sound ? 1 : 0)
+    engineAudio.current.setVolume(sound ? 1 - (speed / maxSpeed) * 2 : 0)
     accelerateAudio.current.setVolume(sound ? (speed / maxSpeed) * 2 : 0)
+    rpmTarget = Math.pow(speed / maxSpeed, 1.5) + (controls.boost ? 0.55 : 0.5)
+    accelerateAudio.current.setPlaybackRate(THREE.MathUtils.lerp(accelerateAudio.current.playbackRate, rpmTarget, delta * 20))
```

Comment:
```suggestion
    accelerateAudio.current.setPlaybackRate(MathUtils.lerp(accelerateAudio.current.playbackRate, rpmTarget, delta * 20))
````
MathUtils is already being imported from Three as constant.